src/directives/multiSelectAutocomplete.js

```diff
@@ -4,8 +4,12 @@ angular.module('bahmni.appointments')
     .directive('multiSelectAutocomplete', [function () {
         var link = function ($scope, element) {
             $scope.focusOnTheTest = function () {
-                var autoselectInput = element.find("input.input");
-                autoselectInput[0].focus();
+                const addAppointmentUrl = "#/home/manage/appointments/calendar/new";
```

Comment:
@AlekhyaYalla 
can we move these constants to constants file ??

Reply:
added